src/pages/cli/hosting/hosting.mdx

```diff
@@ -52,6 +52,8 @@ If you're hosting a Single Page Web App (SPA) with routing such as [`react-route
 
 ![SPA redirect](/images/hosting/spa-redirect.png)
 
+The source address should be: `</^[^.]+$|\.(?!(css|gif|ico|jpg|js|png|txt|svg|woff|woff2|ttf|map|json)$)([^.]+$)/>`, the target address should be `/index.html`, and the type is 200 (Rewrite).
```

Comment:
Could this be linked to https://docs.aws.amazon.com/amplify/latest/userguide/redirects.html#redirects-for-single-page-web-apps-spa instead? That way, if changed, it will only need updated in the Console docs.